rust/operator-binary/src/restart_controller/statefulset.rs

```diff
@@ -110,6 +110,7 @@ pub async fn start(client: &Client) {
                 (),
             ),
         ),
+        Config::default(),
```

Comment:
I just chose default. I'm not sure of the consequences of this.
The other operators don't use the `applier` function.

[`Config` struct docs](https://docs.rs/kube/latest/kube/runtime/struct.Config.html)

Reply:
@nightkr can you please have a look?

Reply:
Reading through https://github.com/kube-rs/kube/pull/1265 and https://github.com/kube-rs/kube/issues/1247 this will keep the current behavior. Using the default Config also enables us to take advantage of future improvements and e.g. an increased default debounce time period. So LGTM.